app/partials/referencedByView.html

```diff
@@ -0,0 +1,30 @@
+<record-address-bar ng-if="showAddressBar"></record-address-bar>
+<!-- TODO: search bar disabled for alpha because not coded yet -->
+<search-bar ng-if="false" ng-controller="recordSearchController"></search-bar>
+<div ng-if="record" class="column-wrapper">
+    <div class="tree-columns">
+        <span ng-repeat-start="column in columns" class="column column-{{::$index}}" ng-class="{'sortable' : column.canSort}" ng-click="toggleSort(column)">
+            {{::column.label}}
+            <span class="sort-arrow" ng-if="column.label === sort.column" title="Elements are sorted by this column in {{sort.reverse ? 'ascending' : 'descending'}} order.">
```

Comment:
If sorting isn't working then this shouldn't be here.  Should we get sorting working?

I think in this case we can do the sorting in JavaScript directly, or just not have sorting at all.  I personally don't see much value in sorting the Referenced By view.

Reply:
Actually, sorting by signature is kind of useful...  So we should support sorting via JS somehow.

Reply:
Sorting from JS will require loading all of the data we're sorting on for the `nodes`.  This shouldn't take a noticeable amount of time unless we're dealing with 500+ nodes, which I don't think is the case for most records.  Definitely something that will need to be tested though.

Reply:
:+1: